docs/hub/how-to-inference.md

```diff
@@ -0,0 +1,44 @@
+# How to integrate the Inference API in your library
+
+The Inference API provides fast inference for your hosted models. Integration with the Hub allows you to access the Inference API programmatically. This guide will show you how to make calls to the Inference API from your library. For more detailed information, refer to the [Inference API documentation](https://api-inference.huggingface.co/docs/python/html/index.html).
+
+Begin by creating an instance of the `InferenceAPI` with a specific model repository ID:
+
+```python
+>>> from huggingface_hub.inference_api import InferenceApi
+>>> inference = InferenceApi(repo_id="bert-base-uncased", token=API_TOKEN)
```

Comment:
Should we explain what `API_TOKEN` is and where to get it from? (I know it's obvious to us, but I'm putting myself in the shoes of a first-time visitor)

Reply:
Good idea, I'll add a brief explanation of the `API_TOKEN` and where to get it.